charts/nri-kube-events/templates/_helpers.tpl

```diff
@@ -29,3 +33,13 @@ allowPrivilegeEscalation: false
 readOnlyRootFilesystem: true
 {{- end -}}
 {{- end -}}
+
+
+
+{{- /*  */ -}}
+{{- define "nri-kube-events.agentConfig" -}}
+is_forward_only: true
+http_server_enabled: true
+http_server_port: 8001
+{{ include "newrelic.common.agentConfig.defaults" . }}
```

Comment:
if I am not wrong these comes from a PR already merged, can we rebase?

Reply:
Rebased :)